src/java.base/share/classes/java/lang/Process.java

```diff
@@ -99,23 +116,26 @@
  * process and for the communication streams between them.
  * The resources to control the process and for communication between the processes are retained
  * until there are no longer any references to the Process or the input, error, and output streams
- * or readers, or they have been closed.
+ * or readers, or they have been closed. The Process {@linkplain Process#close close} method closes
+ * all the streams and terminates the process to release the resources. Using try-with-resources to
+ * {@linkplain ProcessBuilder#start() start} the process can ensure the process
+ * is terminated when the try-with-resources block exits.
  *
  * <p>The process is not killed when there are no more references to the {@code Process} object,
  * but rather the process continues executing asynchronously.
  * The process implementation closes file descriptors and handles for streams
  * that are no longer referenced to prevent leaking operating system resources.
  * Processes that have terminated or been terminated are monitored and their resources released.
  *
- * <p>Streams should be {@code closed} when they are no longer needed, to avoid delaying
+ * <p>Streams should be closed when they are no longer needed, to avoid delaying
```

Comment:
Since there's no way to link to `Reader.close`, `Writer.close`, `InputStream.close` and `OutputStream.close` at the same time, consider linking to some of those from `@see` tags.